.editorconfig

```diff
@@ -6,3 +6,8 @@ indent_style = space
 indent_size = 2
 insert_final_newline = true
 trim_trailing_whitespace = true
+editor.codeActionsOnSave = {
```

Comment:
What does this do? Are you sure we need this? I usually prefer that the editor doesn't do this automatically be instead through an explicit CLI command and git add, git commit.

Reply:
Yes I agree, this should be an explicit action we trigger on the CLI